surf-for-type-safety/package.json

```diff
@@ -0,0 +1,80 @@
+{
+  "name": "surf-example-app",
+  "private": true,
+  "version": "0.0.0",
+  "license": "Apache-2.0",
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc && vite build",
+    "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
+    "preview": "vite preview",
+    "_fmt": "prettier 'frontend/**/*.ts'",
+    "fmt": "npm run _fmt -- --write",
+    "move:init": "node ./scripts/init",
+    "move:test": "node ./scripts/move/test",
+    "move:compile": "node ./scripts/move/compile",
+    "move:publish": "node ./scripts/move/publish",
+    "move:upgrade": "node ./scripts/move/upgrade",
+    "move:get-abi": "node ./scripts/move/get_abi"
+  },
```

Comment:
I would suggest using the boilerplate package.json and not DA/FA as some dependencies not needed here

Reply:
im using boilerplate package.json's script, i copied these script because i have a non empty contract.

Reply:
I mean use the dependencies in the boilerplate template, since most of the dependencies you use now are for Irys which I dont think you use in this example

Reply:
if i use the package.json from boilerplate, the node script to run aptos cli won't work.

Reply:
i can try delete them one by one and see which one are not needed

Reply:
ok I see what you mean, the boilerplate is missing those scripts - should be added.

I think it would be easier for you to copy-past the boilerplate dependencies and devDependencies arrays and add dependencies as needed (probably will need to add some shadcn dependencies)